src/char/char.cpp

```diff
@@ -265,10 +276,8 @@ AString mmo_char_tostr(struct CharPair *cp)
             p->hair, p->hair_color, p->clothes_color,
             p->weapon, p->shield, p->head_top, p->head_mid, p->head_bottom,
             p->last_point.map_, p->last_point.x, p->last_point.y,
-            p->save_point.map_, p->save_point.x, p->save_point.y, p->partner_id);
-
-    // memos were here (no longer supported)
```

Comment:
@wushin you could've taken karma instead; I would much rather use a variable for that